api/api_auth.py

```diff
@@ -0,0 +1,34 @@
+import logging
```

Comment:
I think you could actually call this file `__init.py__`, and then wherever you use it you can `import api` instead of `import api.api_auth`. Up to you if you want to be bothered.